docworks-dts/lib/multiple-files/repoCreator.js

```diff
@@ -12,12 +12,13 @@ const addIndentation = (statment) => `\t${statment}`
 
 const createModulesFilesMap = ({ services, run$wFixer }) => {
 	const servicesMap = createHierarchicalServicesMap(services)
-	return Object.keys(servicesMap).reduce((filesMap, moduleName) => {
-		const rootService = servicesMap[moduleName]
+	return Object.keys(servicesMap).reduce((filesMap, moduleId) => {
+		const rootService = servicesMap[moduleId]
 		if (isEmptyModule(rootService)) return filesMap
+    const moduleName = is$w(moduleId) || !rootService.displayName ? moduleId : rootService.displayName
```

Comment:
why do u need to ask about $w here ? i think it's redundant because $w root service shouldn't have displayName..

Reply:
I know, but it's in order to prevent future bugs, display name not relevant for $w

Reply:
I can remove it if u want to

Reply:
not that critical, ur call